apps/fluent-tester/src/FluentTester/TestComponents/RadioGroup/RadioGroupTest.tsx

```diff
@@ -10,29 +10,43 @@ const basicRadioGroup: React.FunctionComponent<{}> = () => {
     console.log(key);
   };
 
+  const [selectedKey, setSelectedKey] = React.useState('C');
+
+  const onChange2 = React.useCallback(
```

Comment:
nit: can fix or leave it as is but I think this example makes it hard to see the difference between a controlled and uncontrolled RadioGroup. Perhaps you can recreate the partner's scenario on a simple level to reflect its use? I'm not sure what the partner's scenario is here so that may or may not be an easy thing to do.

Reply:
The partner's scenario is the following: Multiple people working on the same document in Excel. The value of the RadioGroup might change asynchronously. If someone changes the value of the RadioGroup, and another person has the RadioGroup visible at the same time, they need to be able to manually change the RadioGroup value. I'm not sure there's a good way to recreate this exact scenario.

There's no functional difference between controlled/uncontrolled from the user's POV, so I think it would be a little difficult to showcase the difference here. The difference comes in the code and how the developer plans to use the RadioGroup. Since there are many scenarios, I'm not sure choosing just one will add any value. I also just took a look at [Fabric's RadioGroup page](https://developer.microsoft.com/en-us/fluentui#/controls/web/choicegroup) and they have the same example for both controlled/uncontrolled. I'm going to leave as is for now :)

Reply:
Interesting. And yeah that's a good point, no worries then :) thanks for implementing this!